klaviyo-react-native-sdk.podspec

```diff
@@ -1,10 +1,9 @@
 require "json"
 
 package = JSON.parse(File.read(File.join(__dir__, "package.json")))
-folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
 
 Pod::Spec.new do |s|
-  s.name         = "klaviyo-react-native-sdk"
+  s.name         =  package["name"]
```

Comment:
using the name from the package just to avoid having it in two places. Not related to the fix.